lib/ts/CryptoHash.h

```diff
@@ -70,7 +72,11 @@ union CryptoHash {
   uint64_t
   fold() const
   {
-    return u64[0] ^ u64[1];
+    return u64[0] ^ u64[1]
+#if CRYPTO_HASH_SIZE >= 256
```

Comment:
```
+#ifdef TS_ENABLE_FIPS
 +// #include "ts/SHA256.h"
 +#define CRYPTO_HASH_SIZE (256 / 8)
 +#else
 +// #include "ts/ink_code.h"
 +#define CRYPTO_HASH_SIZE (128 / 8)
```
it seems like CRYPTO_HASH_SIZE is never large than 256.

Reply:
Changed to ==.

Reply:
Sorry my mistake, It seems like CRYPTO_HASH_SIZE is 32 if TS_ENABLE_FIPS is true.
So we need to change the `#if CRYPTO_HASH_SIZE >= 256` to `#if CRYPTO_HASH_SIZE >= 32`.

Reply:
```
#include <stdio.h>
#include <stdlib.h>

#define TS_ENABLE_FIPS 1

#ifdef TS_ENABLE_FIPS
// #include "ts/SHA256.h"
#define CRYPTO_HASH_SIZE (256 / 8)
#else
// #include "ts/ink_code.h"
#define CRYPTO_HASH_SIZE (128 / 8)
#endif

int
main()
{
  printf("size: %d\n", CRYPTO_HASH_SIZE);
}
```
```
size: 32
```

Reply:
Fixed. thanx! I changed it so that the compiler will complain if the #ifdef isn't complete.